DataFormats/HLTReco/src/classes_def.xml

```diff
@@ -65,7 +79,14 @@
    <version ClassVersion="12" checksum="2301242282"/>
   </class>
   <class name="std::vector<trigger::TriggerEventWithRefs::TriggerFilterObject>"/>
-  <class name="trigger::TriggerEventWithRefs" ClassVersion="12">
+  <class name="trigger::TriggerEventWithRefs" ClassVersion="19">
+   <version ClassVersion="19" checksum="1258968436"/>
+   <version ClassVersion="18" checksum="742506204"/>
+   <version ClassVersion="17" checksum="4163362996"/>
+   <version ClassVersion="16" checksum="1580092994"/>
+   <version ClassVersion="15" checksum="3347721344"/>
+   <version ClassVersion="14" checksum="1580092994"/>
+   <version ClassVersion="13" checksum="1828447080"/>
    <version ClassVersion="12" checksum="3347721344"/>
```

Comment:
Why do we need several generations of added class versions?
From your intermediate tests? We should just have the most recent ones as the intermediate ones are not used.